charts/incubator/automatic-music-downloader/questions.yaml

```diff
@@ -1,28 +1,5 @@
 # Include{groups}
-portals:
-  web_portal:
```

Comment:
@Ornias1993 
IIRC, scale wants the `portals` to be there?
So probably we have to do `portals: {}`